File: Tugas_Praktikum_Pemograman_1_Muhammad_Aufa_Fitrianda_2310817210013/Modul_4/Bahasa_C/Prak403_2310817210013_MuhammadAufaFitrianda.c
#include <stdio.h>

void main(){
    int angka1, angka2;
    
    printf("Masukan Dua Angka Anda\t: ");
    scanf("%d %d",&angka1, &angka2);

    if (angka1 > angka2) {

        for (int i = angka1, j = angka2; i > angka2; i--,j++) {
            printf("%d %d - ",i, j);
        }
    } 

    else {
        
        for (int i = angka1, j = angka2; i < angka2; i++, j--) {
            printf("%d %d - ",i, j);
        }
    }

    printf("%d %d",angka2, angka1);
}

File: Tugas_Praktikum_Pemograman_1_Muhammad_Aufa_Fitrianda_2310817210013/Modul_4/Bahasa_Python/Prak403_2310817210013_MuhammadAufaFitrianda.py
angka1, angka2 =  map(int, input("Masukan Dua Angka Anda\t: ").split())

i = angka1
j = angka2

if angka1 > angka2:
    for i in range(i, j, -1):
        print(f"{i} {j} -", end=" ")
        j += 1
else:
    for i in range(i, j):
        print(f"{i} {j} -", end=" ")
        j -= 1

print(angka2,angka1)